src/Backend/Core/Installer/Data/locale.xml

```diff
@@ -10361,6 +10361,24 @@
         <translation language="pt"><![CDATA[URL]]></translation>
         <translation language="pl"><![CDATA[URL]]></translation>
       </item>
+      <item type="label" name="CanonicalURL">
+        <translation language="nl"><![CDATA[canonical URL]]></translation>
+        <translation language="en"><![CDATA[canonical URL]]></translation>
+        <translation language="hu"><![CDATA[canonical URL]]></translation>
+        <translation language="ru"><![CDATA[canonical URL]]></translation>
+        <translation language="it"><![CDATA[canonical URL]]></translation>
+        <translation language="zh"><![CDATA[canonical URL]]></translation>
+        <translation language="de"><![CDATA[canonical URL]]></translation>
+        <translation language="fr"><![CDATA[canonical URL]]></translation>
+        <translation language="es"><![CDATA[canonical URL]]></translation>
+        <translation language="lt"><![CDATA[canonical URL]]></translation>
+        <translation language="uk"><![CDATA[canonical URL]]></translation>
+        <translation language="sv"><![CDATA[canonical URL]]></translation>
+        <translation language="el"><![CDATA[canonical διεύθυνση URL]]></translation>
```

Comment:
mistake in the translation?
Translations always fall back to english if the translation can't be found.
So I'd rather have the translations missing than containing an untranslated string